src/m365/spo/commands/orgassetslibrary/orgassetslibrary-add.ts

```diff
@@ -54,6 +62,10 @@ class SpoOrgAssetsLibraryAddCommand extends SpoCommand {
       {
         option: '--cdnType [cdnType]',
         autocomplete: ['Public', 'Private']
+      },
+      {
+        option: '--orgAssetType  [orgAssetType ]',
+        autocomplete: ['ImageDocumentLibrary', 'OfficeTemplateLibrary', 'OfficeFontLibrary']
```

Comment:
We should add a validator for these values to ensure that we don't allow folks to specify any value and consider it something else in case they don't match.

Reply:
Do you want me to do the same for the `cdnType`? It's not in the scope but I think it would be good if we do the same for the `cdnType` option

Reply:
Good idea! Let's do it

Reply:
Hey @nicodecleyre, sorry for the late reply. PR looks good. I'll mark it as draft so that you can add a validator for cdnType and we'll get it in then, ok? Thank you so much! 👏